OpenRA.Game/Map/Map.cs

```diff
@@ -766,8 +772,10 @@ public void MakeDefaultPlayers()
 			var firstRace = Rules.Actors["world"].Traits
 				.WithInterface<CountryInfo>().First(c => c.Selectable).Race;
 
-			if (!Players.ContainsKey("Neutral"))
-				Players.Add("Neutral", new PlayerReference
+			var players = GetPlayers();
+
+			if (!players.ContainsKey("Neutral"))
+				players.Add("Neutral", new PlayerReference
```

Comment:
It would be very useful for the new editor if this MakeDefaultPlayers method was also moved out of Map.  If you follow my suggestion about creating a new class for manipulating PlayerReferences then that would be a natural place to move this.